src/components/App/App.js

```diff
@@ -60,32 +154,113 @@ function App() {
     return () => document.removeEventListener("mousedown", closePopupOnRemoteClick);
```

Comment:
The overlay handler doesn't work.

Please, correct it


```js

      if (e.target.classList.contains('popup')) {
        closeAllPopups();
      }
```

Reply:
Done